src/modules/variables.js

```diff
@@ -0,0 +1,5 @@
+export const form = document.querySelector('#form');
+export const warningLabel = document.querySelector('.warningLabel');
+export const refreshScores = document.querySelector('#refreshScores');
+export const scoreInput = document.querySelector('#scoreInput');
+export const nameInput = document.querySelector('#nameInput');
```

Comment:
# Optional:
- I would advise you to keep the number of changes/updates to the DOM as low as possible, they are very expensive for the browser.